docker-compose/docker-compose-tracing.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.7'
+
+services:
+
+  # Tempo runs as user 10001, and docker compose creates the volume as root.
+  # As such, we need to chown the volume in order for Tempo to start correctly.
+  init:
+    image: grafana/tempo:latest
+    user: root
+    entrypoint:
+      - "chown"
+      - "10001:10001"
+      - "/var/tempo"
+    volumes:
+      - tempo-data:/var/tempo
```

Comment:
Does this work properly?

Because this container is restarted every time by `docker-compose up` after it exits.
It is a bit weird to have a service that is supposed to exit after doing its job

Reply:
Add `restart: on-failure`?

Reply:
This has been taken from the git repo mentioned in the docs:
https://grafana.com/docs/tempo/latest/getting-started/docker-example/#clone-the-tempo-repository-and-start-docker

And yes, it works. It does restart this container everytime I do an up, but it has not bothered me.

Reply:
apparently, we don't need the init